blocked-edges/4.17.0-OAuthServerDownIfSpaceInIDPName.yaml

```diff
@@ -0,0 +1,8 @@
+to: 4.17.0
+from: .*
+url: https://issues.redhat.com/browse/AUTH-550
+name: OAuthServerDownIfSpaceInIDPName
+message: |-
+  The oauth-server is down if there are two or more identity providers configured on the cluster with type htpasswd, keystone, ldap, or basic-authentication and at least one of their names contains a white space.
```

Comment:
nit: "The oauth-server is down if..." -> "The OAuth server may crash-loop if...", just to be a bit more specific in what "down" looks like?